src/election.jl

```diff
@@ -41,14 +41,37 @@ function count_votes(name::String, election::Election)::DistrictScore
         vote_totals = sum(election.vote_counts[district, :])
         vote_shares = election.vote_counts[district, :] ./ vote_totals
         election.vote_shares[district, :] = vote_shares
-        # create Dict matching party to vote count & share
-        vote_pairs = Dict{String, NamedTuple}()
-        for i in 1:length(party_names)
-            party_votes = election.vote_counts[district, i]
-            party_share = election.vote_shares[district, i]
-            vote_pairs[party_names[i]] = (vote_count=party_votes, vote_share=party_share)
-        end
-        return vote_pairs
```

Comment:
Returning a `Dict` that stores all vote count/vote shares for every party in this particular district. Now, in the new code, this is done separately by `vote_count` and `vote_share`, and only for the parties that the user requests!